src/rust/ide/src/constants.rs

```diff
@@ -16,7 +16,7 @@ pub const SERIALIZE_TO_JSON_EXPRESSION:&str = "x -> x.json_serialize";
 pub const PROJECT_MANAGER_ENDPOINT:&str = "ws://127.0.0.1:30535";
 
 /// Default project name used by IDE on startup.
-pub const DEFAULT_PROJECT_NAME:&str = "Untitled";
+pub const DEFAULT_PROJECT_NAME:&str = graph_editor::component::project_name::DEFAULT_PROJECT_NAME;
```

Comment:
In my opinion, those two constants have different roles
1. In ide crate this is the name we create/open UnnamedUntitled project.
2. Here it is just the default string in "ProjectName" widget. It can be even empty, the controllers should set proper name by themself, not rely on the widget defaults.

Therefore I opt for keeping two constants separately. They can be even different - that's better, we will see when controllers forget to set the proper project name.